app/components/Snackbar/index.js

```diff
@@ -127,6 +137,8 @@ class Snackbar extends React.Component {
   }
 
   render() {
+    if (this.state.isWaiting)
+      return null;
```

Comment:
Small nit. Try not to use this style of if statement (statement in the following line without curly braces). It is notorious for some nasty bugs (eg: https://embeddedgurus.com/barr-code/2014/03/apples-gotofail-ssl-security-bug-was-easily-preventable/)

Reply:
Use either in the same line which doesn't cause ambiguity or add curly braces.